.gitlab-ci.yml

```diff
@@ -68,7 +69,9 @@ link_checks_preview:
     URL: ${PREVIEW_DOMAIN}
     DISALLOW_BOTS: "1"
   script:
-    - node /etc/node_modules/link-checker/dd-link-checker.js -e preview -c ${CI_PROJECT_DIR}/local/etc/link-check-config.js
+    - node /etc/node_modules/link-checker/dd-link-checker.js --env preview --configFilePath ${CI_PROJECT_DIR}/local/etc/link-check-config.js
```

Comment:
@zbayoff Do we want to make these `yarn run check-links` or `npm run check-links` like on corp?

Reply:
We should do that, but on Docs we're doing the weird, copy node_modules to the `etc` folder, and not using the Gitlab cache. So `yarn run check-links` returns an error saying it can't find the file `link-checker/dd-link-checker.js`

Reply:
ah gotcha, yea we can address that separately.